### PR TITLE
sys-pc-tool 6.33 (new cask)

### PR DESCRIPTION
**Pull request by IvoElferink**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/s/syride-sys-pc-tool.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  version "6.33"
+  sha256 "509e6b833c7fc907c3c0989704b941acba6f9d13abe5f0c20834ee4d77eb0c4f"
+
+  url "https://www.syride.com/downloads/SYS_PC_Tool_setup_mac-#{version}.pkg"
+  name "Syride SYS PC Tool Software"
+  desc "Necessary for downloading flights, customizing screens, upgrading instrument"
+  homepage "https://www.syride.com/"
+
+  livecheck do
+    skip "No version information available"
```
Comment (daeho-ro) [inline Casks/s/syride-sys-pc-tool.rb:11]:
```suggestion
    url "https://www.syride.com/en/software"
    regex(%r{href="/downloads/SYS[._-]PC[._-]Tool[._-]setup[._-]mac[._-]v?(\d+(?:\.\d+)+)\.pkg}i)
```

File: Casks/s/syride-sys-pc-tool.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    regex(%r{href="/downloads/SYS[._-]PC[._-]Tool[._-]setup[._-]mac[._-]v?(\d+(?:\.\d+)+)\.pkg}i)
+  end
+
+  auto_updates true
+  depends_on macos: ">= :sierra"
+
+  pkg "SYS_PC_TOOL_setup_mac-#{version}.pkg"
+
+  uninstall pkgutil: [
+              "/Applications/SYS-PC-Tool.app",
```
Comment (daeho-ro) [inline Casks/s/syride-sys-pc-tool.rb:21]:
```suggestion
```

File: Casks/s/syride-sys-pc-tool.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+              "com.syride.syspctool",
+            ],
+            delete:  [
+              "/Applications/SYS-PC-Tool.app",
+              "com.syride.syspctool",
```
Comment (daeho-ro) [inline Casks/s/syride-sys-pc-tool.rb:26]:
```suggestion
```

File: Casks/s/syride-sys-pc-tool.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            ]
+
+  zap trash: "~/Library/Saved Application State/Syride.SYS-PC-Tool.savedState"
```
Comment (daeho-ro) [inline Casks/s/syride-sys-pc-tool.rb:29]:
```suggestion
zap trash: [
  "~/.syride",
  "~/Library/Saved Application State/Syride.SYS-PC-Tool.savedState",
  "~/syride",
]
```

File: Casks/s/syride-sys-pc-tool.rb (hunk 1 above)
Comment (IvoElferink) [reply Casks/s/syride-sys-pc-tool.rb]: Thanks, updated livecheck

File: Casks/s/syride-sys-pc-tool.rb (hunk 2 above)
Comment (IvoElferink) [reply Casks/s/syride-sys-pc-tool.rb]: Thanks, updated layout/alignment as well

File: Casks/s/syride-sys-pc-tool.rb (hunk 3 above)
Comment (IvoElferink) [reply Casks/s/syride-sys-pc-tool.rb]: Thanks, updated layout/alignment as well

File: Casks/s/syride-sys-pc-tool.rb (hunk 4 above)
Comment (IvoElferink) [reply Casks/s/syride-sys-pc-tool.rb]: Thanks, updated whole block as suggested

**Comment by daeho-ro** [comment]
Thank you @IvoElferink !
Can you squash your commits with a message same as the title of this PR?

File: Casks/s/syride-sys-pc-tool.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
+cask "syride-sys-pc-tool" do
+  version "6.33"
+  sha256 "509e6b833c7fc907c3c0989704b941acba6f9d13abe5f0c20834ee4d77eb0c4f"
+
+  url "https://www.syride.com/downloads/SYS_PC_Tool_setup_mac-#{version}.pkg"
+  name "Syride SYS PC Tool Software"
```
Comment (khipp) [inline Casks/s/syride-sys-pc-tool.rb:6]:
```suggestion
  name "SYS PC Tool"
```

File: Casks/s/syride-sys-pc-tool.rb [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  desc "Necessary for downloading flights, customizing screens, upgrading instrument"
```
Comment (khipp) [inline Casks/s/syride-sys-pc-tool.rb:7]:
```suggestion
  desc "Software for Syride instruments"
```

File: Casks/s/syride-sys-pc-tool.rb (hunk 5 above)
Comment (IvoElferink) [reply Casks/s/syride-sys-pc-tool.rb]: Thanks, updated the name, but left Syride in it as it's more clear

File: Casks/s/syride-sys-pc-tool.rb (hunk 6 above)
Comment (IvoElferink) [reply Casks/s/syride-sys-pc-tool.rb]: Thanks, done!

File: Casks/s/syride-sys-pc-tool.rb [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  sha256 "509e6b833c7fc907c3c0989704b941acba6f9d13abe5f0c20834ee4d77eb0c4f"
+
+  url "https://www.syride.com/downloads/SYS_PC_Tool_setup_mac-#{version}.pkg"
+  name "Syride SYS PC Tool"
+  desc "Software for Syride instruments"
+  homepage "https://www.syride.com/"
+
+  livecheck do
+    url "https://www.syride.com/en/software"
+    regex(%r{href="/downloads/SYS[._-]PC[._-]Tool[._-]setup[._-]mac[._-]v?(\d+(?:\.\d+)+)\.pkg}i)
```
Comment (khipp) [inline Casks/s/syride-sys-pc-tool.rb:12]:
```suggestion
    url "http://www.syride.com/downloads/SYS_PC_Tool/webconfig-para4.xml"
    regex(/SYS[._-]PC[._-]Tool[._-]setup[._-]mac[._-]v?(\d+(?:[.-]\d+)+)\.pkg/i)
    strategy :xml do |xml, regex|
      url = xml.elements["//sysui_config[@machine='mac']/file"]&.text&.strip
      match = url.match(regex) if url

      match[1]
    end
```

This updates the `livecheck` to use the URL of the in-app updater.

Comment (daeho-ro) [reply Casks/s/syride-sys-pc-tool.rb:12]: Oh, I couldn't find even I use the proxyman, are you using pro version? Then I should consider that.

Comment (IvoElferink) [reply Casks/s/syride-sys-pc-tool.rb]: Wow, I learn that I have to learn a few more things! Thought to do a pretty simple addition ;) Thanks for the insights and suggestions! Very helpful and committed as well.

And then I have to find out the previous request how to squash the commits in this PR, I understand what it does, but don't know yet the steps to perform the squash.

Comment (daeho-ro) [reply Casks/s/syride-sys-pc-tool.rb]: @IvoElferink 

You can reset and re-commit, push.
```
git reset --mixed master
git add Casks/s/syride-sys-pc-tool.rb
git commit -m "syride-sys-pc-tool 6.33 (new cask)"
git push -f
```

Comment (IvoElferink) [reply Casks/s/syride-sys-pc-tool.rb]: Thanks! Done as well

Comment (khipp) [reply Casks/s/syride-sys-pc-tool.rb]: @daeho-ro I'm using the free version, but there’s a little trick that @bevanjkay discovered. Sometimes, you can improve your results by starting the app directly from the binary. I'll send you the link to the description on Slack.

**Review by daeho-ro — APPROVED** [review]

**Review by daeho-ro — APPROVED** [review]

**Pull request merged**